db/import.go

```diff
@@ -218,12 +218,21 @@ func (db *Database) importDoc(docid string, body Body, isDelete bool, existingDo
 
 		// If there's a filter function defined, evaluate to determine whether we should import this doc
 		if db.DatabaseContext.Options.ImportOptions.ImportFilter != nil {
-			shouldImport, err := db.DatabaseContext.Options.ImportOptions.ImportFilter.EvaluateFunction(body)
-			if err != nil {
+			var shouldImport bool
+			var importErr error
+
+			if len(body) == 0 && isDelete {
```

Comment:
Thinking about this further, if isDelete and len(body) > 0, we should probably set body[BodyDeleted]=true.

Reply:
Make sense, this is revised in latest revision.